.github/dependabot.yml

```diff
@@ -21,6 +21,8 @@ updates:
         versions: [">=16.0.0"]
       # we ignore TS as a part of quarterly dependency updates.
       - dependency-name: "typescript"
+
+      - dependency-name: "mongodb"
```

Comment:
I can revert but we don't update the driver quarterly, so I added it to the ignore list.

Reply:
no, please keep! ty!